addons/mail/models/mail_mail.py

```diff
@@ -201,6 +245,21 @@ def _postprocess_sent_message(self, success_pids, failure_reason=False, failure_
             self.browse(mail_to_delete_ids).sudo().unlink()
         return True
 
+    @api.model
+    def _add_inherited_fields(self):
```

Comment:
Linting: would define it after create/unlink, in the "CRUD" section (quite low level model :D )